src/rebuild/scan.c

```diff
@@ -390,7 +390,7 @@ rebuild_object_insert(struct rebuild_tgt_pool_tracker *rpt, uuid_t co_uuid,
 }
 
 #define LOCAL_ARRAY_SIZE	128
-#define NUM_SHARDS_STEP_INCREASE	10
+#define NUM_SHARDS_STEP_INCREASE	64
```

Comment:
Is this increase due to more heavyweight collective?

Reply:
no, just because I see from another unrelated log, the 10 is too smaller cause some cases retry multiple times. so just piggyback on this PR.